components/next_steps_view/steps/complete_profile_step/complete_profile_step.tsx

```diff
@@ -64,7 +66,10 @@ export default class CompleteProfileStep extends React.PureComponent<Props, Stat
     private handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
         let fullNameError;
         if (!event.target.value) {
-            fullNameError = Utils.localizeMessage('next_steps_view.complete_profile_step.nameCannotBeBlank', 'Your name can’t be blank');
+            fullNameError = Utils.localizeMessage('next_steps_view.complete_profile_step.fullNameCannotBeBlank', 'Your full name cannot be blank');
+        }
+        if (event.target.value.length > MAX_FULL_NAME_LENGTH) {
```

Comment:
Should we either null check here or do and `else if` here to avoid a null reference error?
```suggestion
        if (event.target.value?.length > MAX_FULL_NAME_LENGTH) {
```

Reply:
You're right. I'll use the else if